hazelcast/src/main/java/com/hazelcast/cp/internal/datastructures/spi/blocking/ResourceRegistry.java

```diff
@@ -57,7 +57,10 @@
  * @param <R> concrete type of the resource
  */
 public abstract class ResourceRegistry<W extends WaitKey, R extends BlockingResource<W>>
-        implements LiveOperationsTracker, DataSerializable {
+        implements DataSerializable {
+
+    private static final long OPERATION_TIMEOUT_EXTENSION_MS = TimeUnit.SECONDS.toMillis(5);
+    private static final long NO_WAIT_KEY_DEADLINE = -1;
```

Comment:
I think it's better to use `Long.MAX` as `NO_WAIT_KEY_DEADLINE`. That way, there'll be no need to check `deadline == NO_WAIT_KEY_DEADLINE` equality.